salt/transport/__init__.py

```diff
@@ -308,3 +308,36 @@ def send(self, load, tries=3, timeout=60):
             return self._uncrypted_transfer(load, tries, timeout)
         else:  # for just about everything else
             return self._crypted_transfer(load, tries, timeout)
+
+class LocalChannel(Channel):
+    """
+    Local channel for testing purposes
+    """
+    def __init__(self, opts, **kwargs):
+        self.opts = opts
+        self.kwargs = kwargs
+        self.tries = 0
+
+    def send(self, load):
+
+        if self.tries ==0 :
+            print "load", load
```

Comment:
No print statements, please. If we need to see data, please use the appropriate logging functions.